### DISCUSSION TITLE
Projects beta - unable to save views

### DISCUSSION THREAD

**Discussion by jesfer** [community/community#5682] [Projects and Issues] [answered]
If I try to create a new view or save changes to an existing view i get the error message "Sorry, something went wrong".

This started happening about an hour ago and happens consistently ever since.

**Comment by ArthurMialon** [comment]
Hey 👋 

We got the same issue on our new Project Beta.
It seems like all requests to `projects/beta/<ID>/**` send 404 Not Found error.

By the way, we automated some tasks from Github Action via `gh api <GRAPH_QL>` and it seems working well.

Issue seems "only" appeared from the UI. 🤔

**Comment by ArthurMialon** [comment, marked as answer]
It seems fixed on my side today. What about you @jesfer ?

**Reply by jesfer** [reply]
Yup, working for me as well :)
